arcus_orientation.md

```diff
@@ -709,44 +710,52 @@ There are two primary kinds of problems that could occur:
+![Warning message which reads "You are about to enable the internet for your lab. Changing the setting to Internet Enabled puts your data at a greater security risk. You can reduce risks by keeping the internet disabled in your lab and using the Install Tool when installing packages. If you choose to proceed, this action will restart your lab with the new setting. All your data is stored and will be available upon restart. Would you like to proceed?"](media/internet_enabled_warning.png)<!-- style = "max-width: 400px;"-->
 
-In the following sections, we'll review more specific details about how to work with a variety of tools while using Internet Disabled Mode. 
+In the following sections, we'll review more specific details about how to work with a variety of tools while using Internet Disabled mode. 
 
 ### Using R with Internet Disabled
 
 Most functions in R run locally -- once you install the package they're in, they're saved on your machine (or in this case, in your Arcus Lab) for you to use whenever you like, without requiring you to have an internet connection. 
 If you already have R code written for your analysis, chances are all of that code will continue to work in exactly the same way whether or not you have internet disabled. 
 
-We've also set up your Arcus Lab to allow some common remote connections in a secure way, so you can have internet disabled in your lab but still maintain access to certain secure services. 
+We've also set up your Arcus Lab to allow some common remote connections in a secure way, so you can keep your lab in Internet Disabled mode but still maintain access to certain secure services. 
 Here are some things you can **always** access from R, even with internet disabled: 
 
 - Your Arcus data (for example, using commands like `bq_table_download()` from the `bigrquery` package)
 - Repositories on CHOP's Enterprise GitHub (for example, using `clone`, `push`, and `pull`)
 - Opening external links from within RStudio, such as clicking a link for a vignette from within help documentation in R
 
-However, there are a few important R functions that require an internet connection to work (especially [install.packages()](#installing-and-updating-packages-in-r)), potentially posing a security risk for your data. 
+However, there are a few important R functions that require an internet connection to work (especially `install.packages()`), potentially posing a security risk for your data. 
 To provide peace of mind for researchers working in Arcus, we provide a secure method for you to install R packages and download other necessary files without having to enable internet. 
```

Comment:
```suggestion
To provide peace of mind for researchers working in Arcus, the Install Tool lets you install R packages and download other necessary files without having to enable internet. 
```

Reply:
```suggestion
To provide peace of mind for researchers working in Arcus, we provide a secure method for you to install R packages and download other necessary files without having to enable internet: the Install Tool. 
```

I like that the sentence says in words what the Install Tool is, but agree that using the name is also helpful to make it clear that the tool is what we're talking about. How about combining both?